src/Scrapers/Pipeline/Types/ScrapeConfig.ts

```
@@ -0,0 +1,111 @@
+}
+
+/** Configuration for fetching the account/card list. */
+interface IAccountsFetchConfig<TRaw> {
+  /** HTTP method for accounts endpoint. */
+  readonly method: HttpMethod;
+  /** URL path (appended to api.base). */
+  readonly path: string;
+  /** POST body data (for POST method). Empty for GET. */
+  readonly postData: Record<string, string>;
+  /** Extract account IDs + balances from raw API response. */
+  readonly mapper: (raw: TRaw) => readonly IRawAccount[];
```

ESLint

[error] 'raw' is defined but never used.

(no-unused-vars)